snyk-monitor/values.yaml

```diff
@@ -148,6 +148,9 @@ sysdig:
 strategy:
   type: RollingUpdate
 
+initContainers:
+  enabled: false
```

Comment:
It needs to be on by default, otherwise, it's a breaking change!

Reply:
Oops 🙈